Makefile

```diff
@@ -399,3 +399,19 @@ cluster-deploy:
 	IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000 \
 	REGISTRY_NAMESPACE=openshift-storage \
 	$(MAKE) deploy
+
+# Security Analysis
+SEVERITY_THRESHOLD ?= medium
+SNYK_ORG ?= 81de31f3-6dff-46ff-af37-664e272a9fe3
+
+.PHONY: analyze-code
+analyze-code:
```

Comment:
Nit: analyze is a little too generic for my taste in a already highly scoped such as the makefile. Maybe `analyze-vulnerabilities-code` and the likes?

Reply:
I hate long make commands, how about `vuln-scan-code` etc.

Reply:
also works for me!